src/app/pages/DashboardPage/Nodes.tsx

```diff
@@ -36,7 +35,7 @@ export const Nodes: FC = () => {
       <Tooltip
         arrow
         placement="top"
-        title={hasActiveNodes ? t('nodes.tooltip') : t('nodes.unknown')}
+        title={t('nodes.tooltip')}
```

Comment:
Why not simply set `hasActiveNodes` to true?

Reply:
If we did that, it would be easier to revert later. (Less change.)

Reply:
Good point, I will bring it back.